.github/workflows/deploy.yml

```diff
@@ -22,10 +25,16 @@ jobs:
         run: npm ci
 
       - name: Generate
-        run: npm run build
-
+        run: npx nuxi generate
+      - name: Check generated contents
+        run: |
+          pwd
+          ls -la
+          cd .output/public
+          pwd
+          ls -la
```

Comment:
Are these statements necessary in the final deployment? Or are they troubleshooting leftovers?

Reply:
Ah yes, they're more for trouble shooting. If it's ok, I would like to leave them in anyway for now, since we might still run into some issues. If it all works out then we can remove them, I think

Reply:
Sounds good! Perhaps add an issue to remove these in due time?